web/modules/custom/hoeringsportal_content_blocks/src/Plugin/Block/MeetingWarning.php

```diff
@@ -71,6 +71,9 @@ public function build() {
     $node = $this->routeMatch->getParameter('node');
     if (isset($node) && $node->field_content_state->value) {
       $config['content_state'] = $node->field_content_state->value;
+      if ($node->field_public_meeting_cancelled->value > 0) {
+        $config['content_state'] = 'cancelled';
+      }
```

Comment:
Shouldn't we actually update the `content_state` (to `cancelled`) value when a meeting is cancelled? This may break filtering on the overview page (or make it work; do we want to list cancelled meetings?).

Reply:
If we introduce a new content state "cancelled" we need to alter the views. According to PMs cancelled meetings should just appear under active or finished depending on whether their original date is passed or not. So the views will become a lot more messy i think